packages/styles/framework/src/utilities/mixins/_mixins.scss

```
@@ -155,20 +210,59 @@
+    background-color: getColor('background');
     text-align: center;
+    align-items: center;
+    font-size: 14px;
 
-    input:disabled,
-    input:disabled:checked {
-      & ~ label.#{$type}-label {
-        @include disabledRules;
+    &[aria-disabled="true"] {
+      @include disabledRules;
+      filter: none;
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)